iOS/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -212,7 +285,16 @@ extension CourseListViewController: CoreDataCollectionViewDataSourceDelegate {
             ])
         }
 
-        return NSCompoundPredicate(andPredicateWithSubpredicates: subPredicates)
+        let searchTextPredicate = NSCompoundPredicate(andPredicateWithSubpredicates: subPredicates)
+
+        if let activeSearchFilters = self.searchFilterViewController?.activeFilters {
+            let subpredicates = activeSearchFilters.map { filter, selectedOptions in filter.predicate(forSelectedOptions: selectedOptions) }
```

Comment:
Probably rename that to `filterSubPredicates`? You have 2 variables names the same, but different case, and I got confused.

Reply:
Done